sematic/examples/lightning_resnet/pipeline.py

```diff
@@ -0,0 +1,162 @@
+
+    ## Inputs
+    - **config**: Configuration with compute and model parameters for training
+    - **data_config**: Configuration for training data and data loading
+
+    ## Output
+    The best checkpoint produced during training
+    """
+    cluster_config = SimpleRayCluster(
+        n_nodes=config.n_workers, node_config=config.worker
+    )
+
+    # You can also use the built-in pytorch lightning checkpointing
+    # with s3 if you are willing to install some extra dependencies
+    checkpointer = SematicCheckpointIO(
+        s3_location=config.checkpoint_location / context().run_id
+    )
+
+    # we want the driver for the training to run on the
+    # small ray cluster we're creating.
+    @ray.remote(num_cpus=config.worker.cpu, num_gpus=config.worker.gpu_count)
+    def call_train_classifier_from_ray():
+        logging.basicConfig(level=logging.INFO)
+        strategy_compute_kwargs = dict(
+            # -1 because 1 worker is being used to drive the training.
+            num_workers=config.n_workers - 1,
+            num_cpus_per_worker=config.worker.cpu,
+            use_gpu=config.worker.gpu_count > 0,
+        )
+        return train_classifier(
+            config, data_config, strategy_compute_kwargs, checkpointer
+        )
+
+    with RayCluster(config=cluster_config):
+        return ray.get(call_train_classifier_from_ray.remote())
+
+
+@sematic.func(inline=False)
+def evaluate(
+    checkpoint: Checkpoint, config: EvaluationConfig, data_config: DataConfig
+) -> EvaluationResults:
+    """# Evaluate a model checkpoint for a ResNet model.
+
+    ## Inputs
+    - **checkpoint**: A checkpoint for a trained model
+    - **config**: Configuration for compute for evaluation
+    - **data_config**: Configuration for the data to be used during evaluation
+
+    ## Output
+    Summary of some statistics resulting from the evaluation.
+    """
+    cluster_config = SimpleRayCluster(
+        n_nodes=config.n_workers, node_config=config.worker
+    )
+    checkpointer = SematicCheckpointIO(s3_location=checkpoint.prefix)
+
+    # we want the driver for the eval to run on the
+    # small ray cluster we're creating.
+    @ray.remote(num_gpus=config.worker.gpu_count)
+    def call_evaluate_classifier_from_ray():
+        logging.basicConfig(level=logging.INFO)
+        strategy_compute_kwargs = dict(
+            num_workers=config.n_workers - 1,
+            num_cpus_per_worker=config.worker.cpu,
+            use_gpu=config.worker.gpu_count > 0,
+        )
+        return evaluate_classifier(
+            checkpoint, config, data_config, strategy_compute_kwargs, checkpointer
+        )
+
+    with RayCluster(config=cluster_config):
+        return ray.get(call_evaluate_classifier_from_ray.remote())
+
+
+@sematic.func(inline=True)
+def bundle_results(
+    final_checkpoint: Checkpoint,
+    evaluation_results: EvaluationResults,
+) -> PipelineResults:
+    """# Combine the results into a single data structure.
```

Comment:
```suggestion
    """# Combine the results into a single data structure
```

Reply:
section header

Reply:
Yes, the suggestion is to remove the period. Please do so everywhere.